schemas/2016-06-01/Microsoft.RecoveryServices.json

```diff
@@ -0,0 +1,124 @@
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2016-06-01",
+            "2016-05-01",
+            "2015-12-15",
+            "2015-11-10",
+            "2015-08-15",
+            "2015-08-10",
+            "2015-06-10",
+            "2015-03-15"
+          ]
+        },
+        "sku": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/Sku"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Required. Gets or sets the sku type."
+        },
+        "location": {
+          "type": "string",
+          "description": "Required. Gets or sets the location of the resource. This will be one of the supported and registered Azure Geo Regions (e.g. West US, East US, Southeast Asia, etc.). The geo region of a resource cannot be changed once it is created, but if an identical geo region is specified on update the request will succeed."
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater than 128 characters and value no greater than 256 characters."
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/RecoveryServicesPropertiesCreateParameters"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "sku",
+        "location",
+        "properties"
+      ],
+      "description": "Microsoft.RecoveryServices/vaults"
+    }
+  },
+  "definitions": {
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "enum": [
+                "RS0"
+              ]
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Gets or sets the sku name. Required for vault creation, optional for update. Possible values include: 'RS0'"
+        }
+      },
+      "tier": {
```

Comment:
if "tier" is a property of "sku", it should be under "properties", not the same level, the "properties" in ln82 is a schema object property, not name of property.